OTAcceleratorCore/OTOneToOneCommunicator.m

```diff
@@ -279,6 +307,27 @@ - (void)publisher:(OTPublisherKit *)publisher didFailWithError:(OTError *)error
     }
 }
 
+- (void)publisher:(nonnull OTPublisherKit *)publisher streamCreated:(nonnull OTStream *)stream {
+    [[LoggingWrapper sharedInstance].logger logEventAction:KLogActionStartPublishing
+                                                 variation:KLogVariationSuccess
+                                                completion:nil];
+    if (stream.videoType == OTStreamVideoTypeScreen ) {
+        [[LoggingWrapper sharedInstance].logger logEventAction:KLogActionStartScreensharing
+                                                     variation:KLogVariationSuccess
+                                                    completion:nil];
+    }
```

Comment:
Under screen sharing, it will log both. Should we put KLogActionStartPublishing to the else branch?